pages/linux/ebuild.md

```diff
@@ -0,0 +1,35 @@
+# ebuild
+
+> A low level interface to the Gentoo Portage system.
+
+- Create or update the package manifest:
+
+`ebuild {{path/to/file.ebuild}} manifest`
+
+- Clean the temporary build directories for the build file:
+
+`ebuild {{path/to/file.ebuild}} clean`
+
+- Fetch sources if they do not exist:
+
+`ebuild {{path/to/file.ebuild}} fetch`
+
+- Extract the sources to a temporary build directory:
+
+`ebuild {{path/to/file.ebuild}} unpack`
+
+- Compile the extracted sources:
+
+`ebuild {{path/to/file.ebuild}} compile`
+
+- Install the package to a temporary install directory:
```

Comment:
I guess this tool isn't for installing things for real?

Reply:
If you run `qmerge` then it does install the package to the live filesystem. The actual `install` command just installs them in a sandboxed container.

Reply:
Doesn't `live filesystem`  refer to an in-memory filesystem that's wiped when the machine shuts down though?

Reply:
Sorry for the late reply, I didn't actually know that. :thinking:

Reply:
Neither do I! I'm just going by the terminology I'm familiar with - i.e. a "live CD" boots into a "live session", which has a "live file system" - and everything gets wiped on shutdown 'cause it's all stored in RAM.